internal/pkg/template/templates/overrides/yamlpatch/cfn.patches.yml

```diff
@@ -0,0 +1,15 @@
+# Delete the task role resource
+- op: remove
+  path: /Resources/TaskRole
+
+# Add a service connect alias
+- op: add
+  path: /Resources/Service/Properties/ServiceConnectConfiguration/Services/0/ClientAliases/-
+  value:
+    Port: !Ref TargetPort
+    DnsName: yamlpatchiscool
+
+# Replace the task role in the task definition
+- op: replace
+  path: /Resources/TaskDefinition/Properties/TaskRoleArn
+  value: arn:aws:iam::123456789012:role/MyTaskRole
```

Comment:
Should we comment out the entire file, so that when a customer runs `copilot deploy` immediately after the `override` command we don't suddenly delete the task role 😝

Reply:
it's a feature not a bug